ta/mk/lib.mk

```diff
@@ -0,0 +1,77 @@
+# Get the dir of the ta-dev-kit, requires make version 3.81 or later
```

Comment:
This file is a patched copy of `ta/mk/ta_dev_kit.mk` resulting in duplicated code that is hard to keep up to date. This has to be addressed in some way.